src/I18n/Time.php

```diff
@@ -45,6 +45,22 @@ class Time extends Carbon implements JsonSerializable
     protected static $_toStringFormat = [IntlDateFormatter::SHORT, IntlDateFormatter::SHORT];
 
     /**
+     * The format to use when when converting this object to json
+     *
+     * The format should be either the formatting constants from IntlDateFormatter as
+     * described in (http://www.php.net/manual/en/class.intldateformatter.php) or a pattern
+     * as specified in (http://www.icu-project.org/apiref/icu4c/classSimpleDateFormat.html#details)
+     *
+     * It is possible to provide an array of 2 constants. In this case, the first position
+     * will be used for formatting the date part of the object and the second position
+     * will be used to format the time part.
+     *
+     * @var mixed
+     * @see \Cake\I18n\Time::i18nFormat()
+     */
+    protected static $_jsonEncodeFormat = "yyyy-MM-dd'T'HH:mm:ssxx";
```

Comment:
That's true, but if you think about this, you will have `setToStringFormat` and `setJsonEncodeFormat` methods, but they will have different syntax. Don't you think they should have the same arguments format?

And @lorenzo ... I am sorry for not understanding "missing the timezone part"... I read it on cellphone and responded via email. I am a bit confused, since tests worked on my machine and now I see, they are broken in travis. I'll fix that, but @markstory please try to give me a hint, if I should use the format for `formatz method or for `i18nformat` method. Thanks